addons/s3/models.py

```diff
@@ -78,29 +87,43 @@ def set_folder(self, folder_id, auth):
         self.folder_name = '{} ({})'.format(folder_id, bucket_location)
         self.save()
 
-        self.nodelogger.log(action='bucket_linked', extra={'bucket': str(folder_id)}, save=True)
```

Comment:
@mfraezz I was wondering what you think about this obviously I want to avoid migrating the logs, I figure if this is purely additive (the bucket name will stay the same string.) we can add path a new thing without causing discontinuous historical data. I feel just adding something to the logs like this is appropriate if it's in `extra`.